datadog/resource_datadog_security_monitoring_rule.go

```diff
@@ -1433,3 +1443,22 @@ func resourceDatadogSecurityMonitoringRuleDelete(ctx context.Context, d *schema.
 
 	return nil
 }
+
+func resourceDatadogSecurityMonitoringRuleCustomizeDiff(ctx context.Context, diff *schema.ResourceDiff, meta interface{}) error {
+	if validate, ok := diff.GetOkExists("validate"); !ok || (ok && !validate.(bool)) {
```

Comment:
```suggestion
	if validate, ok := diff.GetOkExists("validate"); !ok || !validate.(bool) {
```

Reply:
Thanks, not sure when this was changed, i had this while coding :(